src/lib/AccountStore.js

```diff
@@ -116,6 +121,45 @@ class AccountStore {
         });
     }
 
+    /**
+     * @param {string} id
+     * @param {Uint8Array} password
+     * @returns {Promise<Key?>}
+     */
+    async get(id, password) {
+        /** @type {KeyRecord?} */
+        const keyRecord = await this._get(id);
+        if (!keyRecord) return null;
+
+        const secret = await Nimiq.Secret.fromEncrypted(new Nimiq.SerialBuffer(keyRecord.secret), password);
+        return new Key(secret, keyRecord.hasPin);
+    }
+
+    /**
+     * @param {string} id
+     * @returns {Promise<KeyInfo?>}
+     */
+    async getInfo(id) {
+        const keyRecord = await this._get(id);
+        if (!keyRecord) return null;
+        return KeyInfo.fromObject(keyRecord, KeyStore.isEncrypted(keyRecord), keyRecord.defaultAddress);
+    }
+
+    /**
+     * @param {string} id
+     * @returns {Promise<KeyRecord?>}
+     * @private
+     */
+    async _get(id) {
+        const db = await this.connect();
+        if (!db) return null;
+        const transaction = db.transaction([AccountStore.ACCOUNT_DATABASE], 'readonly');
+        const request = transaction.objectStore(AccountStore.ACCOUNT_DATABASE).get(id);
+        const accountRecord = await KeyStore._requestToPromise(request, transaction);
+        if (!accountRecord) return null;
+        return KeyStore.accountRecords2KeyRecords([accountRecord])[0];
```

Comment:
AccountStore is a legacy class and should hopefully be removed in the future. I think there is nothing wrong with calling static helper functions on the KeyStore, instead of a higher-order class.